src/models/NotificationSetting.js

```diff
@@ -17,4 +17,5 @@ module.exports = (sequelize, DataTypes) => sequelize.define('NotificationSetting
   serviceId: { type: DataTypes.STRING, allowNull: false },
   name: { type: DataTypes.STRING, allowNull: false },
   value: { type: DataTypes.STRING, allowNull: false },
+  topicOld: { type: DataTypes.STRING, allowNull: false },
```

Comment:
@sachin-maheshwari do we need to keep the old topic name? I mean immediately after running the migration script, we can drop this column.

Reply:
@vikasrohit  No. This is just temporary, will clean later.